internal/datanode/mock_test.go

```diff
@@ -38,7 +38,6 @@ import (
 	"github.com/milvus-io/milvus/internal/proto/etcdpb"
 	"github.com/milvus-io/milvus/internal/proto/rootcoordpb"
 	"github.com/milvus-io/milvus/internal/storage"
-	s "github.com/milvus-io/milvus/internal/storage"
```

Comment:
why not remove this file directly and use mockery replace it ?

Reply:
it contains many codes like test utils, not only mock object